apps/wix/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Wix.com
```

Comment:
Do you want to update the year to `2024` here?

Reply:
Per the legal guidelines at Wix, it should be 2015, as it represents the year copyright was applied.